docs.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Docs
+---
+<ul class="nav nav-tabs" style="margin-bottom: 20px;">
+    <!-- <li><a href="#">Overview</a></li> -->
```

Comment:
Yes, absolutely, these links will be built from a data object rather than statically defined. This is merely an artifact of my work-process: focusing purely on layout to the relative exclusion of data structure until the next phase.